wger/gym/templates/gym/partial_user_list.html

```diff
@@ -14,6 +15,16 @@
 });
 </script>
 
+<div class="tab">
+    <button class="tablinks" onclick="displayMembers(event, 'active_users')" id="defaultOpen">Active Users</button>
+    <button class="tablinks" onclick="displayMembers(event, 'inactive_users')">Inactive Users</button>
```

Comment:
Use `Deactivated Users` instead of `Inactive Users`

Reply:
@AGMETEOR I have changed this as well